pkg/controller/mongodb/replicaset_controller_test.go

```diff
@@ -656,7 +657,7 @@ func setStatefulSetReadyReplicas(t *testing.T, c k8sClient.Client, mdb mdbv1.Mon
 
 // loadTestFixture will create a MongoDB resource from a given fixture
 func loadTestFixture(yamlFileName string) (mdbv1.MongoDBCommunity, error) {
-	testPath := fmt.Sprintf("testdata/%s", yamlFileName)
+	testPath := path.Join(e2eutil.TestdataDir, yamlFileName)
```

Comment:
The testdata dir was referenced in many places, and it changed since `go test` uses the directory of the test file as the current, not the root directory of the project. I refactored it along with the deploy directory into constants in e2eutil.